psydac/ddm/cart.py

```diff
@@ -61,7 +61,7 @@ class CartDecomposition():
         (optional: default is MPI_COMM_WORLD).
 
     """
-    def __init__( self, npts, pads, periods, reorder, comm=MPI.COMM_WORLD ):
+    def __init__( self, npts, pads, periods, reorder, comm=MPI.COMM_WORLD, nprocs=None, reverse_axis=None ):
```

Comment:
What if we need to reverse more than one axis? Does that never happen?

Reply:
it doesn't happen we always want to reverse one at a time , if we have an L shape patches we build the first patch then the second is reversed by one axis of the first aftet that the third is reversed by one axis of the second even though that the third is reversed by 2 axes of the first